### PR TITLE
feat(Comment): add size prop to Group

### LINKED ISSUE #1311: Add `size` prop to Comment.Group
It is missing in SUI core docs, but is available in the CSS.

```jsx
<Comment.Group className="small" />
```

Should be accomplished with 

```jsx
<Comment.Group size="small" />
```

### PR DESCRIPTION
**Pull request by tarang9211**
Fixed #1311

### CODE REVIEW

File: src/views/Comment/CommentGroup.js [hunk 1]
```diff
@@ -57,6 +59,9 @@ CommentGroup.propTypes = {
   /** Comments can hide extra information unless a user shows intent to interact with a comment. */
   minimal: PropTypes.bool,
 
+  /** Comments can have different sizes*/
+  size: PropTypes.oneOf(['mini', 'tiny', 'small', 'large', 'big', 'huge', 'massive']),
```
Comment (layershifter) [inline src/views/Comment/CommentGroup.js]: We use `SIZES` const for definitions, [example](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/collections/Message/Message.js#L94).

File: test/specs/views/Comment/CommentGroup-test.js [hunk 2]
```diff
@@ -8,4 +8,7 @@ describe('CommentGroup', () => {
   common.propKeyOnlyToClassName(CommentGroup, 'collapsed')
   common.propKeyOnlyToClassName(CommentGroup, 'minimal')
   common.propKeyOnlyToClassName(CommentGroup, 'threaded')
+  common.propValueOnlyToClassName(CommentGroup, 'size', [
+    'mini', 'tiny', 'small', 'large', 'big', 'huge', 'massive',
```
Comment (layershifter) [inline test/specs/views/Comment/CommentGroup-test.js]: ^

**Review by layershifter — CHANGES_REQUESTED** [review]
And we need add `size` to typings.

**Comment by tarang9211** [comment]
@layershifter alright, fixed up. let me know.

**Review by layershifter — APPROVED** [review]

**Comment by levithomason** [comment]
Fantastic, we just need a doc example for "Group Size" now.  Here are some existing group size doc examples for comparison:

[Examples/elements/Button/GroupVariations/ButtonExampleGroupSize.js](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Examples/elements/Button/GroupVariations/ButtonExampleGroupSize.js)
[Examples/elements/Label/Groups/LabelExampleGroupSize.js](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Examples/elements/Label/Groups/LabelExampleGroupSize.js)
[Examples/views/Statistic/Types/StatisticExampleGroupSize.js](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/app/Examples/views/Statistic/Types/StatisticExampleGroupSize.js)

The comment group size should be listed in `Comment/Variations/CommentExampleGroupSize.js`

**Comment by tarang9211** [comment]
@levithomason I've added doc site examples as per your request. Only thing I noticed was that the props for size was listed this way:

screen shot 2017-02-15 at 1 24 14 am

**Comment by tarang9211** [comment]
@levithomason I think this should wrap up this fix!

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Thanks again guys!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.65.0`

**Pull request merged**